packages/cloud_functions/cloud_functions/android/src/main/java/io/flutter/plugins/firebase/cloudfunctions/CloudFunctionsPlugin.java

```diff
@@ -49,9 +49,9 @@ public void onMethodCall(MethodCall call, final Result result) {
           functions.useFunctionsEmulator(origin);
         }
         HttpsCallableReference httpsCallableReference = functions.getHttpsCallable(functionName);
-        Number timeoutMilliseconds = call.argument("timeoutMilliseconds");
```

Comment:
This change comes as a result of discussion in the [first PR](https://github.com/FirebaseExtended/flutterfire/pull/1826) in this series, where we discovered that the iOS plugin and the Dart code were using the named parameter `timeoutMicroseconds`. As a result, the Android plugin would always use the default timeout even if a timeout was specified, since looking for parameter `timeoutMilliseconds` would always return `null`. I haven't yet searched to see if there's already a bug filed for this behavior. If there is, then this change will also address that bug.

Reply:
There is already an issue filed for this: #95